reference/pipeline/stages/index.md

```diff
@@ -169,6 +169,11 @@ Deploy a Kubernetes manifest YAML/JSON file.
 ### Find Artifacts From Resource
 Find artifacts from a Kubernetes resource.
 
+### Patch (Manifest)
+Update an already existing Kubernetes resource in place using the [Kubernetes patch operation](https://kubernetes.io/docs/tasks/run-application/update-api-object-kubectl-patch/). Spinnaker can update the resource without knowing the entire resource manifest i.e. you have to specify only the portion of the manifest you want to update. For instance, the patch stage can be used to add a label or update a sidecar container image for a set of resources. It can also be used to implement a red-black or "rainbow" deployment strategy for Kubernetes by first deploying a new ReplicaSet and then patching the fronting service's selectors to point to the new ReplicaSet.
```

Comment:
I don't know about the rainbow strategy. I can't tell if you're presenting it as an alternate term for red/black (which, I hope not; it's already a synonym for blue/green I think!), or if it's just another strategy. If it is different, I would...
s/implement a red-black or "rainbow" deployment strategy/implement a red/black deployment strategy or "rainbow" deployment strategy/

(Note that it's red/black not red-black.)

Reply:
Unrelated nit: would you mind splitting up this paragraph into multiple lines, at the 80-100 character mark? (The way other long blocks of text are formatted in this doc). It makes it easier to review specific lines.

Reply:
It is basically the same as red/black or blue/green. So, I've removed rainbow and kept just red/black to keep things consistent.

Reply:
It's not entirely the same, and this approach has specific consequences that make it different. For example, in the prior red/black strategy one could have one load balance manage multiple replica sets during normal deployments. Using this strategy, at most one replica set can be managed.

Reply:
Split into two paragraphs.

Also, removed red/black and added "rainbow" back in with a link to an external article. Will replace the link with the guide I'm writing on it when complete.

Reply:
Looks good now!